openml/config.py

```diff
@@ -123,6 +153,13 @@ def _setup():
             'server load reasonable'
         )
 
+    # Verbosity levels as defined (https://github.com/openml/OpenML/wiki/Client-API-Standards)
+    # don't match Python values directly:
+    verbosity_map = {0: logging.WARNING, 1: logging.INFO, 2: logging.DEBUG}
+    console_log_level = verbosity_map[_defaults['verbosity']]
```

Comment:
I'm afraid that you need to change this mapping here...

Reply:
Oops. Sorry. Though I shouldn't change the mapping here, but rather change the default values. At least, I assume we want to use the API Client defined levels instead of the Python levels (and values)?